plugins/Eq/EqControlsDialog.cpp

```diff
@@ -208,6 +208,14 @@ EqControlsDialog::EqControlsDialog( EqControls *controls ) :
 	hpBtnGrp->addButton( hp24Button );
 	hpBtnGrp->addButton( hp48Button );
 	hpBtnGrp->setModel( &m_controls->m_hpTypeModel,false);
+
+	//Oversample
+	PixmapButton * oversampleButton = new PixmapButton( this, NULL );
+	oversampleButton->setActiveGraphic( PLUGIN_NAME::getIconPixmap( "oversampleon" ) );
+	oversampleButton->setInactiveGraphic(  PLUGIN_NAME::getIconPixmap( "oversampleoff" ) );
+	oversampleButton->setCheckable(true);
```

Comment:
You should add `oversampleon.png` and `oversampleoff.png` to make the button usable.

Reply:
Whoops!  Completely forgot to add those.